openml/flows/sklearn_converter.py

```diff
@@ -0,0 +1,466 @@
+from collections import OrderedDict, defaultdict
+import importlib
+import inspect
+import json
+import json.decoder
+import six
+import warnings
+import sys
+
+import numpy as np
+import scipy.stats.distributions
+import sklearn.base
+import sklearn.model_selection
+# Necessary to have signature available in python 2.7
+from sklearn.utils.fixes import signature
+
+from .flow import OpenMLFlow
+from ..exceptions import OpenMLRestrictionViolated
+
+MAXIMAL_FLOW_LENGTH = 1024
+
+
+if sys.version_info >= (3, 5):
+    from json.decoder import JSONDecodeError
+else:
+    JSONDecodeError = ValueError
+
+
+class SklearnToFlowConverter(object):
```

Comment:
I might be missing something but I still don't get this design.
We are providing a ``serialize`` function (or method right now) that takes an sklearn estimator and returns an ``OpenMLFlow``, right? Why is ``create_flow_from_model`` needed?
If someone wanted to do something like that for another library, say ``keras``, they'd write their own function ``serialize_keras`` that takes a keras model and produces an ``OpenMLFlow``, right?